operatorcourier/courier_api.py

```diff
@@ -0,0 +1,36 @@
+"""
+operatorcourier.courier_api module
```

Comment:
IMHO `operatorcourier.api` is enough, is is clear form import path that API belongs to operator courier

Reply:
Addressed in latest commit.